xla/service/gpu/tests/gemm_rewrite_test.cc

```diff
@@ -1184,6 +1275,7 @@ class LegacyCublasGemmRewriteTest : public GemmRewriteTest {
   DebugOptions GetDebugOptionsForTest() override {
     DebugOptions debug_options = GemmRewriteTest::GetDebugOptionsForTest();
     debug_options.set_xla_gpu_enable_cublaslt(false);
+    debug_options.set_xla_gpu_simplify_all_fp_conversions(true);
```

Comment:
Why enable this for all tests?

Reply:
cuz mix type gemm test is guarded by this flag, this is not enabled in google internal test somehow. And it seems it is enabled by default when i ran it on my local machine

Reply:
removed. Turns out it is related to filecheck.